app/models/post.rb

```diff
@@ -1,2 +1,15 @@
 class Post < ApplicationRecord
+  has_many :likes, dependent: :destroy
+  has_many :comments, dependent: :destroy
+  belongs_to :author, class_name: 'User'
+
+  def return_recent_comments
+    comment.order(created_at: :desc).limit(5)
+  end
+
+  private
+
+  def update_posts_counter
+    author.increment!(:posts_counter)
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly use `after_save` to call `update_posts_counter` and make the `update_posts_counter` method private. The `after_save` will call this method each time a new post is saved in the database.

Reply:
Noted.